### PR TITLE
Increase Maximum Allowed File Size

### PR DESCRIPTION
**Pull request by mjallday**
Increase max allowed file size for https://github.com/verygood-ops/infra-vault/issues/1142

http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size

```
Syntax:		client_max_body_size size;
Default:	client_max_body_size 1m;
Context:	http, server, location
```
> Sets the maximum allowed size of the client request body, specified in the “Content-Length” request header field. If the size in a request exceeds the configured value, the 413 (Request Entity Too Large) error is returned to the client. Please be aware that browsers cannot correctly display this error. Setting size to 0 disables checking of client request body size.

### CODE REVIEW

File: templates/nginx.conf.j2 [hunk 1]
```diff
@@ -54,11 +54,14 @@ http {
 	# gzip_buffers 16 8k;
 	# gzip_http_version 1.1;
 	# gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
+	
+	# Override maximum file size accepted
+	client_max_body_size size 100m;
```
Comment (max-lobur) [inline templates/nginx.conf.j2]: client_max_body_size 100m;

Comment (max-lobur) [reply templates/nginx.conf.j2]: will that fit Java app limits?

Comment (mjallday) [reply templates/nginx.conf.j2]: it exceeds them but i don't see any issue with that. i picked the same size that cloudflare allows for their non-premium subscriptions.

https://support.cloudflare.com/hc/en-us/articles/201303340-How-can-I-change-the-client-maximum-upload-size-

Comment (max-lobur) [reply templates/nginx.conf.j2]: I see, thanks

Comment (max-lobur) [reply templates/nginx.conf.j2]: fix the syntax issue and it's good. Do you want the new stack or apply this on a current?
Do we need to test how java handles limit exceed or that's good?

**Pull request merged**